Tests/test_synapse_launcher.py

```diff
@@ -0,0 +1,79 @@
+import unittest
+
+import mock
+
+from kalliope.core.Models import Brain
+from kalliope.core.Models.Settings import Settings
+from kalliope.core.SynapseLauncher import SynapseLauncher, SynapseNameNotFound
+
+from kalliope.core.Models import Neuron
+from kalliope.core.Models import Order
+from kalliope.core.Models import Synapse
+
+
+class TestSynapseLauncher(unittest.TestCase):
+    """
+    Test the class SynapseLauncher
+    """
+
+    def setUp(self):
+        # Init
+        neuron1 = Neuron(name='neurone1', parameters={'var1': 'val1'})
+        neuron2 = Neuron(name='neurone2', parameters={'var2': 'val2'})
+        neuron3 = Neuron(name='neurone3', parameters={'var3': 'val3'})
+        neuron4 = Neuron(name='neurone4', parameters={'var4': 'val4'})
+
+        signal1 = Order(sentence="this is the sentence")
+        signal2 = Order(sentence="this is the second sentence")
+        signal3 = Order(sentence="that is part of the third sentence")
+
+        self.synapse1 = Synapse(name="Synapse1", neurons=[neuron1, neuron2], signals=[signal1])
+        self.synapse2 = Synapse(name="Synapse2", neurons=[neuron3, neuron4], signals=[signal2])
+        self.synapse3 = Synapse(name="Synapse3", neurons=[neuron2, neuron4], signals=[signal3])
+
+        all_synapse_list = [self.synapse1,
+                            self.synapse2,
+                            self.synapse3]
+
+        self.brain_test = Brain(synapses=all_synapse_list)
+        self.settings_test = Settings(default_synapse="Synapse3")
+
+    def test_match_synapse1(self):
```

Comment:
We are used to test each method by defining only one test_method. 
You should adapt all the 4 following test into 1 to keep the code consistent.